services/ui-src/src/components/export/ExportedEntityStepCard.tsx

```diff
@@ -0,0 +1,200 @@
+// components
+import { Card } from "components";
+import { Box, Image, Text } from "@chakra-ui/react";
+// utils
+import {
+  AnyObject,
+  EntityShape,
+  HeadingLevel,
+  OverlayModalStepTypes,
+} from "types";
+// assets
+import completedIcon from "assets/icons/icon_check_circle.png";
+import unfinishedIcon from "assets/icons/icon_error_circle.png";
+import { fillEmptyQuarters } from "utils";
+import { ObjectiveProgressEntity } from "components/cards/ObjectiveProgressEntity";
+import { EvaluationPlanEntity } from "components/cards/EvaluationPlanEntity";
+import { FundingSourcesEntity } from "components/cards/FundingSourcesEntity";
+
+export const ExportedEntityStepCard = ({
+  entityIndex,
+  entityTotal,
+  stepType,
+  formattedEntityData,
+  hasBoxShadow,
+  hasBorder,
+  ...props
+}: Props) => {
+  let entityCompleted = false;
+  const entitiesCount = `${entityIndex + 1} / ${entityTotal}`;
+  // any drawer-based field will do for this check
+  let cardContent = {};
+  switch (stepType) {
+    case OverlayModalStepTypes.OBJECTIVE_PROGRESS:
+      entityCompleted = formattedEntityData?.objectiveName;
```

Comment:
`formattedEntityData` is always defined so can remove `?`s with it in this file.